### PR TITLE
add --host command line argument

### LINKED ISSUE #19: make host configurable
On mac every time it reloads there is a firewall alert because it's fixed listen on `0.0.0.0`. Can we add `--host` option to override the interface?

### PR DESCRIPTION
**Pull request by kanekv**
Fixes #19

### CODE REVIEW

**Comment by di** [comment]
Thanks @kanekv!

Can you document this in [this section of the README](https://github.com/GoogleCloudPlatform/functions-framework-python/blob/master/README.md#configure-the-functions-framework), and add a line to the changelog as well?

**Comment by kanekv** [comment]
@di done

**Comment by di** [comment]
@kanekv I'll do the version bump in a separate PR, please add the changelog entry under the "Unreleased" header.

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.1.2] - 2020-02-20
+### Added
+- Add support for `--host` flag ([#20])
```
Comment (di) [inline CHANGELOG.md]: This needs a corresponding reference at the end of this file.

**Comment by kanekv** [comment]
@di looks better now?

**Comment by di** [comment]
Thanks!

**Pull request merged**